recipes/pretextsnapshot/meta.yaml

```diff
@@ -26,6 +26,10 @@ test:
   commands:
     - {{ name }}
 
+extra:
+  skip-lints:
+    - should_not_be_noarch_source
```

Comment:
Adding the `skip-lints` is correct.
Looks like that is a known issue: xref: https://github.com/bioconda/bioconda-utils/issues/598